src/components/common/ColonyActionsTable/ColonyActionsTable.tsx

```diff
@@ -115,7 +125,16 @@ const ColonyActionsTable: FC<ColonyActionsTableProps> = ({
       title={formatText({ id: 'activityFeedTable.table.title' })}
       tableComponent={TableWithMeatballMenu}
       tableProps={tableProps}
-    />
+      headerClassName="w-full"
+    >
+      <Filters
```

Comment:
I am not 100% sure we should use a reusable component for any filters. The logic can be proprietary, so can be the interfaces.
Could we for starters do filters separately for this table and see if any patterns emerge? 
There are different ways you can go about it, but maybe just create a dropdown in this folder and connect each item with appropriate handlers to modify the filter state.